src/components/Profile/components/VictoryPromise/components/VictroyPromiseIcons.tsx

```diff
@@ -0,0 +1,267 @@
+import { Typography } from '@mui/material';
+import { Tooltip } from '@mui/material';
+import { toTitleCase } from 'services/utils';
+import { Colors, VictoryPromiseCategory, VictoryPromiseColor } from 'services/victoryPromise';
+import { gordonColors } from 'theme';
+import styles from './VictoryPromiseIcon.module.css';
+
+const CategoryDescription = {
+  christian_character:
+    'Opportunities encouraging faith formation and its connection to living, learning and leading with others',
+  intellectual_maturity:
+    'Opportunities to extend critical reasoning, deepen understanding, and ignite imagination',
+  lives_of_service:
+    "Opportunities to lend one's strengths and talents with our partners to our neighbors",
+  leadership_worldwide:
+    "Opportunities to develop one's understanding and influence in God's amazing, dynamic and challenging world",
```

Comment:
```suggestion
    'Opportunities encouraging faith formation and its connection to living, learning and leading with others',
  intellectual_maturity:
    'Opportunities to extend critical reasoning, deepen understanding, and ignite imagination',
  lives_of_service:
    'Opportunities to lend one's strengths and talents with our partners to our neighbors',
  leadership_worldwide:
    'Opportunities to develop one's understanding and influence in God's amazing, dynamic and challenging world',
```
I don't mind double quotes if that is what you prefer, but we should be consistent about which quotes we are using in the same object.

Reply:
Prettier did that automatically. As is apparent with GitHub's syntax highlighting, the double quotes are needed on the last two because they contain single quotes. However, Prettier defaults to single quotes when double quotes aren't needed, which is why they're inconsistent. The only way to override that would be to disable Prettier formatting for these lines, which I don't think is a good option.